packages/components/menu/src/menu-button.stories.ts

```diff
@@ -168,7 +180,7 @@ export const WithAvatar: Story = {
   }
 };
 
-export const All: Story = {
+export const Combination: Story = {
```

Comment:
It looks like the example needs to be updated? :)

<img width="306" alt="image" src="https://github.com/user-attachments/assets/14dd5ebf-2bcd-4481-aaf7-6a780f03dd25">

Reply:
Fixed